lib/middleware/request-handler.ts

```diff
@@ -406,11 +412,17 @@ export function elevateRequest (req: Interface.IOurRequest,
         res.write(text + JSON.stringify(data));
     };
 
-    res.sendEnd = (status: string, message: string): void => {
-        var text = util.format('%s%s,%s}',
+    res.sendEnd = (status: string,
```

Comment:
I could go either way personally -- if it's ok to have `correlationIds` embedded in the data, then great, but if that's not optimal, then I'd rather augment `sendEnd` in the short run and maybe have a TODO for improving it later.